web-wallet/src/lib/dusk/components/Mnemonic/Mnemonic.svelte

```diff
@@ -57,11 +57,25 @@
   }
 
   /**
-   * Adds word to the entered phrase if only one suggestion is available
-   * @param {{ key: string }} event
+   * Prevents non-alphabetical characters from being entered
+   * and auto-selects the first suggestion on Enter
+   * if there is only one word available
+   * @param {KeyboardEvent} event
    * @param {string} index
    */
   function handleKeyDownOnAuthenticateTextbox(event, index) {
+    const isAlphabetical = /^[a-zA-Z]+$/;
+
+    if (!isAlphabetical.test(event.key)) {
```

Comment:
Shouldn't this check be after the "enter key" one?

The second one won't fire for the last suggestion as the enter key is not an alphabetical char.

Reply:
`event.key` is `Backspace` for the backspace key and `Enter` for the enter – checked against the `isAlphabetical` regex, both will go through.